packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAdsTableRow/MyAdsTableRow.tsx

```diff
@@ -12,7 +12,7 @@ import DeleteIcon from '../../../../../public/ic-delete.svg';
 import { PopoverDropdown } from '@/components';
 import { ADVERT_TYPE, RATE_TYPE } from '@/constants';
 import { formatMoney } from '@/utils/currency';
-import { AdStatus, AdType, ProgressIndicator } from '../../../components';
+import { AdStatus, AdType, AlertComponent, ProgressIndicator } from '../../../components';
```

Comment:
shouldn't these go under @/components ? also sort these imports

Reply:
no. these aren't the reusable ones.. they are the ones specific to myads..

Reply:
hmmm if its only forMyAds, we should move it under the MyAds folder and create a separate components folder for it 👀

Reply:
yeah. its already under my-ads-> components

Reply:
ahhh got it